ptftplib/dhcpserver.py

```diff
@@ -386,6 +395,9 @@ def main():
     parser.add_option("-a", "--answer-all-dhcp-requests", dest="answer_all_requests",
                       help="Enables DHCP response to all clients, "
                            "default is PXE clients only", action="store_true", default=False)
+    parser.add_option("-n", "--name-servers", dest="name_servers",
```

Comment:
I'm guessing the reason you talk about "between one and three DNS servers" is because that's how many you can pass in in the DHCP response. In this case I recommend you actually enforce that in the command-line argument parsing.

If you switch to a `-n/--name-server` option, you can use the `nargs` parameter to `add_option()` to control how many times that option may be specified on the command-line, resulting in the following usage:

```
$ ... --name-server 8.8.8.8 --name-server 1.1.1.1
```

And this would correctly fail:

```
$ ... -n 8.8.8.8 -n 8.8.4.4 -n 1.1.1.1 -n 1.0.0.1
```

Reply:
I have no problem with the requested changes, however after some testing I have been unable to get OptionParser to behave as you describe here.  Setting `nargs` to X seems to make optparse require X arguments be passed to the flag.

Like `-n 8.8.8.8 8.8.4.4 1.1.1.1`

https://pyfiddle.io/fiddle/702744a1-92e3-4531-b5ca-68aa1aefd7ab/

>If you switch to a `-n/--name-server` option 

I'm not sure what you mean here.  What do I need to switch this to for optparse to work as expected?  

The only reference I found to the proposed flag syntax is in https://docs.python.org/2/library/optparse.html#adding-new-actions

Reply:
I added a commit to enable the desired behavior.  Please let me know if https://github.com/mpetazzoni/ptftpd/pull/18/commits/336e68ae4eea4ccd12b1b705656af02cc0d05e12 is acceptable, or if you have better way to do it through optparse.